media/admin/com_virtuemart/controllers/zasilkovna.php

```diff
@@ -59,14 +59,22 @@ public function save($data = 0)
         /** @var VirtueMartModelZasilkovna $model */
         $model = VmModel::getModel('zasilkovna');
         $currentData = $model->loadConfig();
-        $model->updateConfig(array_replace_recursive($currentData, $data));
+        $message = null;
```

Comment:
tady by se nabízelo vytvořit value object, který bude přijímat message a typ a uvnitř budou definovány typy message, které lze použít.  Sjednotí to tak použití message.